dashboard/utils.py

```diff
@@ -329,23 +336,22 @@ def get_overall_final_grade_for_course(self, course):
         Returns:
            str: the overall final grade
         """
-        if settings.FEATURES.get('USE_COMBINED_FINAL_GRADE', False):
+        best_grade = self.get_best_final_grade_for_course(course)
+        if best_grade is None:
+            return ""
+        if not course.has_exam:
+            return str(round(best_grade.grade_percent))
 
+        if settings.FEATURES.get('USE_COMBINED_FINAL_GRADE', False):
```

Comment:
This may be too late to be helpful, but I was just thinking that we could refactor out this `USE_COMBINED_FINAL_GRADE` feature flag. I don't think we still need to support have that set to False.

Reply:
The code in the `else` is also untested, but it seems like we can just remove it according to this comment. We should also look for other uses of `USE_COMBINED_FINAL_GRADE` and deal with them appropriately

Reply:
@annagav Did you have a chance to look at this?